### PR TITLE
Pull changes from jboothman/Tizen.NET to Samsung/Tizen.NET

### PR DESCRIPTION
**Pull request by jboothman**
To add changes to _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md

### CODE REVIEW

File: _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md [hunk 1]
```diff
@@ -10,50 +10,42 @@ toc_sticky: true
...
 ![][collectionview3]
 
 ## 3. Material Visual
 
-Material Design is an opinionated design system created by Google, that prescribes the size, color, spacing, and other aspects of how views and layouts should look and behave.
+Material Design is an opinionated design system created by Google, which prescribes the size, color, spacing, and other aspects of how views and layouts should look and behave.
 
-Xamarin.Forms provides the way to apply material design across the platforms to controls in Xamarin.Forms apps.
-This is achieved with material renderers, that apply the Material Design rules.
+Xamarin.Forms provides the way to apply material design across the platforms to controls in Xamarin.Forms apps. Material renderers apply the Material Design rules.
```
Comment (JoonghyunCho) [inline _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md]: For the last sentence, isn't it better to keep the original one to be more clear?
"This is achieved with material renderers, that apply the Material Design rules."

File: _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md [hunk 2]
```diff
@@ -10,50 +10,42 @@ toc_sticky: true
 ---
 
 
-`Xamarin.Forms 4.1.0` has been recently released on Nuget, and I'd like to introduce what's new for Tizen on `Xamarin.Forms 4.1.0`. <br/>
-As you already know, many cool new features are added on `Xamarin.Forms 4.0`, and you may be wondered if those features are available on Tizen platform.
+`Xamarin.Forms 4.1.0` has been released on NuGet. This post introduces the new Tizen features on `Xamarin.Forms 4.1.0`. <br/>
+Many new UI features were added on `Xamarin.Forms 4.0`, including Shell, CollectionView and Material Visual, which are supported on the Tizen platform.
```
[the remark this answers is not in the document]
Comment (jboothman) [reply _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md]: True. Let's remove the word Tizen from the second sentence so it reads:  "This post introduces the new features on `Xamarin.Forms 4.1.0`."

File: _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md (hunk 1 above)
Comment (jboothman) [reply _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md]: OK. But please remove the comma - it isn't necessary.

Comment (JoonghyunCho) [reply _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md]: Ok! applied!

File: _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (JoonghyunCho) [reply _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md]: I rewrote the sentences, please check again.

[the remark this answers is not in the document]
Comment (jboothman) [reply _posts/2019-07-15-whats-new-for-tizen-on-forms-410.md]: Looks good - thank you.

**Pull request merged**